src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
First, u don't need directly call 'toString()' method, u have overridden it . and when u call System.out.println(object) it will call object.toString() and print it.

Second, u don't need to create three instances of a class object, u can create Lottery, and call 3 times method getRandomBall();

Third, u need to print it 3 times in the console, instead of copy-paste System.out.println(), u can use for cycle for 3 iterations, inside it System.out.println(lottery.getRandomBall())